panel/views/pages/tree.php

```diff
@@ -53,6 +53,18 @@
                         <div class="page-route truncate mr-2" aria-hidden="true">
                             <span><?= $page->canonicalRoute() ?? $page->route() ?></span>
                         </div>
+                        <?php
+                            $imagePreviewFieldName = $page->scheme()->options()->get('imagePreviewField', '');
+                            $imagePreviewFieldValue = $imagePreviewFieldName == '' ? '' : $page->fields()->get($imagePreviewFieldName);
+                            $imagePreviewFile = $imagePreviewFieldValue == '' ? '' : $page->files()->get($imagePreviewFieldValue);
+                        ?>
```

Comment:
I'd avoid unnecessary variables and prefer `null`s.
```suggestion
                        <?php $imagePreviewField = $page->scheme()->options()->get('imagePreviewField') ?>
```